lib/definitions/open.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "open" {
```

Comment:
I am clues-less here. Is it appropriate for other CLI's? Just let me know and I will move it.